.github/workflows/pull_request.yml

```diff
@@ -169,3 +172,8 @@ jobs:
 
             _Updated at ${{ steps.build-components.outputs.time }} with changes from ${{ steps.build-components.outputs.commit_id }}_
           edit-mode: replace
+          body: |
+            The latest commit did not produce rendering changes.
+
+            _Updated at ${{ steps.build-components.outputs.time }} with changes from ${{ steps.build-components.outputs.commit_id }}_
+          edit-mode: replace
```

Comment:
Is this section just a copy of what's above in lines 170-174?

Reply:
Oh yeah not sure why that happened. Good catch.